src/Common/CosmosClient.ts

```diff
@@ -105,6 +108,13 @@ export function client(): Cosmos.CosmosClient {
   if (configContext.PROXY_PATH !== undefined) {
     (options as any).plugins = [{ on: "request", plugin: requestPlugin }];
   }
+
+  if (userContext.features.enablePriorityBasedThrottling && userContext.apiType === "SQL") {
```

Comment:
we are changing the RP api to use enablePriorityBasedExecution instead of enablePriorityBasedThrottling. Can you please change this?